filebeat/module/nginx/_meta/config.full.yml

```diff
@@ -0,0 +1,33 @@
+- module: nginx
+  # Access logs
+  filesets.access:
+    enabled: true
+
+    # Format. Options are with_plugins and no_plugins
+    #var.pipeline: with_plugins
```

Comment:
I was considering inlining `var` as well,  like we did for `fileset`. That would be the nicest CLI experience but the chances of collisions are higher in there.

The reason I called them "vars" is that they can reference each other. But I think I'm good with `option` as well, have to think some more about it.